catalog-rest-service/src/main/resources/ui/src/pages/services/index.tsx

```diff
@@ -102,13 +117,20 @@ const ServicesPage = () => {
   const [editData, setEditData] = useState<ServiceDataObj>();
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const [searchText, setSearchText] = useState('');
+  const [currentTabTotalCount, setCurrentTabTotalCount] = useState(0);
+
   const [servicesCount, setServicesCount] = useState({
     databaseServices: 0,
     messagingServices: 0,
     dashboardServices: 0,
     pipelineServices: 0,
   });
 
+  const [isLoadingMore, setIsLoadingMore] = useState<boolean>(false);
+  const [status, setStatus] = useState<'initial' | 'waiting' | 'success'>(
```

Comment:
@ShaileshParmar11 you can make a separate type for it.

Reply:
updated